okhttp/src/test/java/okhttp3/KotlinSourceCompatibilityTest.kt

```diff
@@ -946,19 +946,19 @@ class KotlinSourceCompatibilityTest {
     var recordedRequest: RecordedRequest = RecordedRequest(
         "", Headers.of(), listOf(), 0L, Buffer(), 0, Socket())
     recordedRequest = RecordedRequest(null, null, null, 0L, null, 0, Socket())
-    var requestUrl: HttpUrl? = recordedRequest.getRequestUrl()
-    var requestLine: String? = recordedRequest.getRequestLine()
-    var method: String? = recordedRequest.getMethod()
-    var path: String? = recordedRequest.getPath()
-    var headers: Headers? = recordedRequest.getHeaders()
+    var requestUrl: HttpUrl? = recordedRequest.requestUrl
```

Comment:
the source incompatibility offends me here, though I don’t know what the heck we do about it.

Reply:
Fixed.